deploy/quickstart-k8s.yaml

```diff
@@ -53,7 +53,7 @@ roleRef:
 subjects:
   - kind: "ServiceAccount"
     name: everest-admin
-    namespace: percona-everest
+    namespace: everest-system
```

Comment:
Is this SA actually used? It was configured for an unexisting namespace, but nothing is failing bc of this.

Reply:
Oh, I see, thank you. So looks like no additional action needed.